genesis_devtools/tests/unit/test_dependency.py

```diff
@@ -146,3 +148,183 @@ def test_env_path_fetch(
         finally:
             shutil.rmtree("/tmp/genesis_core_test_dir")
             shutil.rmtree("/tmp/___deps_dir")
+
+    def _run_local_path_dep_fetch(
+        self,
+        src_dir: str,
+        target_dir: str,
+        files: tp.List[str],
+        exclude: tp.List[str],
+        dep_config_base: tp.Dict[str, tp.Any],
+    ) -> str:
+        """
+        Function helper for exclude tests set
+        Creates temporary file structures, runs LocalPathDependency.fetch.
+        and return path out.
+        """
+        # Creating all folders and files
+
+        for file_path in files:
+            full_path = os.path.join(src_dir, file_path)
+            os.makedirs(os.path.dirname(full_path), exist_ok=True)
+            with open(full_path, "w") as f:
+                f.write(f"content of {file_path}")
+
+        # Create a target folder
+        os.makedirs(target_dir, exist_ok=True)
+
+        # Add exclude
+        dep_config = dict(dep_config_base)
+        dep_config["exclude"] = exclude
+
+        dep = deps.LocalPathDependency.from_config(dep_config, "/tmp")
+        dep.fetch(target_dir)
+
+        return os.path.join(target_dir, os.path.basename(src_dir))
+
+    def test_exclude_dir_root_lvl(self):
+        # Example 1. Exclude the build file from root
```

Comment:
Let's remove `example X` from code, it's doesn't mean anything without context.


```suggestion
        # Exclude the build file from root
```